src/wwwroot/template-editor/adapterService.js

```diff
@@ -0,0 +1,346 @@
+(function() {
+    'use strict';
+  
+    angular
+      .module('mseditor')
+      .service('adapterService', adapterService);
+  
+      adapterService.$inject = [
+      '$http',
+      '$q'
+    ];
+  
+    function adapterService($http, $q) {
+      var service = {
+        getCampaign: getCampaign,
+        campaignSaveChanges: campaignSaveChanges,
+        campaignSaveChangesAsTemplate: campaignSaveChangesAsTemplate,
+        getCampaignRssPreview: getCampaignRssPreview,
+        deleteImages: deleteImages,
+        getImagesForCampaign: getImagesForCampaign,
+        getPreviewImage: getPreviewImage,
+        getPreviewImageWithPlay: getPreviewImageWithPlay,
+        getSettings: getSettings,
+        refreshMercadoShopsProductServiceToken: refreshMercadoShopsProductServiceToken,
+        getTiendaNubeProducts: getTiendaNubeProducts
+      };
+  
+      return service;
+
+      /**
+       * Return a preview placeholder image
+       * @argument {Object} params - Configuration object
+       * @property {string} params.fileName - The file name and extension
+       * @property {string} params.providerName - The provider name
+       * @property {number} params.idCampaign - The campaign identifier
+       * @property {number} params.idTemplate - The template identifier
+       * @example
+       * // {
+       * //  fileName: 'testFile.jpg',
+       * //  providerName: providerName,
+       * //  idCampaign: 0,
+       * //  idTemplate: 0
+       * // }
+       */
+      function getPreviewImageWithPlay(params) {
+        console.log("getPreviewImageWithPlay");
```

Comment:
What do you think about only log output parameters for non promises results?

Reply:
Well, in fact, I am more interested in input parameters because they are generated by the MSEditor and could be interesting read them in the console. 

Maybe, by the way, we could also include our demo output result, but I am not sure if it is useful or not.

Does it answer your question?

Reply:
Yes, thank you